File: java-sdk/src/main/java/com/blinkpaycard/sdk/BlinkPayCardConfig.java
package com.blinkpaycard.sdk;

public class BlinkPayCardConfig {

    /**
     * 此为测试账号的开发配置，申请正式账号后请替换
     */
    private String appId = "appEqMuvZ088";
    private String mchId = "blEAgkNtLm8";
    private String appKey = "keyW8LNi4GJzyMqEsnjS2ZMw6kkNtDUObxB";

    public String getAppId() {
        return appId;
    }

    public void setAppId(String appId) {
        this.appId = appId;
    }

    public String getMchId() {
        return mchId;
    }

    public void setMchId(String mchId) {
        this.mchId = mchId;
    }

    public String getAppKey() {
        return appKey;
    }

    public void setAppKey(String appKey) {
        this.appKey = appKey;
    }
}

File: php_sdk/example/BlinkPayCard.Config.php
<?php
/**
*
* 实际部署时，请务必保管自己的商户密钥，证书等
* 
**/

class BlinkPayCardConfig
{


	//=======【BlinkPayCard信息配置】=====================================
	/**
     *
	 * TODO: 此为测试账号的开发配置，申请正式账号后请替换
	 * 
	 * APPID：绑定支付的APPID
	 * 
	 * MCHID：商户号
	 * 
	 * KEY：商户支付密钥
	 * 
	 **/


	public function getAppId()
	{
		return "appEqMuvZ088";
	}

	public function getMerchantId()
	{
		return "blEAgkNtLm8";
	}
	
	public function getAppKey()
	{
		return "keyW8LNi4GJzyMqEsnjS2ZMw6kkNtDUObxB";
	}


}
